dm/dm/master/openapi_controller.go

```diff
@@ -443,14 +444,29 @@ func (s *Server) deleteTask(ctx context.Context, taskName string, force bool) er
 	}
 	defer release()
 
+	ignoreCannotConnectError := func(err error) bool {
+		if err == nil {
+			return true
+		}
+		if force && strings.Contains(err.Error(), "connect: connection refused") {
```

Comment:
how about use 
`errors.Is(err, syscall.ECONNREFUSED)`

Reply:
i tried `errors.Is(err, syscall.ECONNREFUSED)` and run inegration test, but it can't ignore `connection refused`. I still don't know the root cause, maybe i will find it out later

Reply:
i think this is because this error is  wrapped by `terror` https://github.com/pingcap/tiflow/blob/e91554504e44d3b5faa1b3c8d777b3306dc964c3/dm/pkg/conn/basedb.go#L117

ok not change here